### DISCUSSION TITLE
can't read properties of null (reading kdftype)

### DISCUSSION THREAD

**Discussion by skyetang** [dani-garcia/vaultwarden#4847] [Q&A]
"When connecting the Chrome extension Bitwarden through self-hosting, I received an error message: 'can't read properties of null (reading kdftype)'. I set up the server myself, and it can be accessed and used normally, with HTTPS enabled."

The command used to set up the server is:
docker run -d --name vaultwarden -v /vw-data/:/data/ --restart unless-stopped -p 8080:80 -e SIGNUPS_ALLOWED=false -e DOMAIN=https://xxx.com vaultwarden/server:latest.

**Comment by Tarosul** [comment]
Same problem here

**Comment by senna1992** [comment]
Second that.


login in from Webbrowser and android App works fine

This is from my docker container's debug log:
`s6-rc: info: service legacy-services successfully started
[2024-08-18 22:50:34.156][request][INFO] GET /
[2024-08-18 22:50:34.156][response][INFO] (web_index) GET / => 200 OK
[2024-08-18 22:50:38.294][request][INFO] POST /
[2024-08-18 22:50:38.294][rocket::server::_][ERROR] No matching routes for POST / application/json.
[2024-08-18 22:50:38.294][rocket::server::_][WARN] Responding with registered (not_found) 404 catcher.
[2024-08-18 22:50:38.294][response][INFO] 404 Not Found`

**Reply by senna1992** [reply]
add: please echeck if your server adress [entered inside of the chrome addon| 
is having "/#login or something similar added automatically.
after having removed that, it worked for me!

**Reply by Mozza7** [reply]
To add onto this, I thankfully had it signed in on a different browser already so could compare.
I entered https://my.domain.com and it auto added /#login as it did for you. On my other it was just "my.domain.com".

After removing https:// and /#login all is well (it is accessible via https, just to note)

**Reply by yonnath** [reply]
thanks it worked for me too

**Comment by dpnoworyta** [comment]
Same story goes to safari Bitwarden extension (tried also in chrome).
In server address I do not have anything typed after URL itself.

**Comment by luisschwab** [comment]
Same issue here. Mobile and desktop apps work fine.

version: v2024.6.2b

**Comment by BlackDex** [comment]
Make sure the URL is correct, including `https://` and not `http://` and no spaces or other strange characters are in-front or behind the URL it self like `#` or whatever.

Saying that the other clients do work correctly mostly indicates the server is configured correctly. But there still might be a misconfiguration at the reverse proxy, or maybe an update of the reverse proxy which enables some security features like ModSecurity or WAF features.

**Reply by dpnoworyta** [reply]
Well everything was working fine just fine.
Still something needed to change - because I did not update my setup since 2 weeks ago (`1.32.0`), thus I'm thinking that browser add-on needed to update itself in that time

**Reply by dpnoworyta** [reply]
hmm, on the other hand - in macos - Bitwarden safari extension is part of Bitwarden desktop client and that one is working correctly...

**Comment by luisschwab** [comment]
The URL is correct, but I do think it's a reverse proxy issue (I just don't know what it is). Apps work when accessing via LAN IP address, but fails when accessing via WAN domain name with HTTPS.

**Reply by luisschwab** [reply]
Is there an example nginx config file?

**Reply by BlackDex** [reply]
https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples

**Reply by dpnoworyta** [reply]
I don't think that is something with reverse-proxy settings and that's because mobile/desktop clients are working correctly (I'm running vaultwarden behind traefik reverse-proxy with ssl serving)

**Reply by tfrajt** [reply]
Hi there, 

I faced similar problem, output error message looked like this:
<img width="370" alt="Screenshot_bitwarden_errmsg" src="https://github.com/user-attachments/assets/be32ee67-846e-4cc5-b1ac-c55ce2afc8fd">

In my case I was able to solved it by removing server URL from all configuration fields of the extension except the first. 
For details of running configuration please see following picture:
<img width="376" alt="Screenshot_bitwarden" src="https://github.com/user-attachments/assets/993b7506-2408-41ed-830b-d4b4e4eff16c">


Hopefully someone finds my "discovery :D" helpfull.


cheers,
Tomas.
